compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -92,7 +92,8 @@ llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
     }
   }
 
-  return os << "{" << "enableReduceSharedMemoryBankConflicts = "
+  return os << "{"
+            << "enableReduceSharedMemoryBankConflicts = "
```

Comment:
Undo this please -- seems like you formatted the whole file and your clang-format is out of sync with the one used by CI